integrations/push-bullet.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Created by idoschachter on 17/07/2016.
+ */
+'use strict';
+
+var Promise = require('bluebird');
+var PushBullet = require('pushbullet');
+
+let key = process.env.PUSHBULLET_KEY;
+let pusher;
+
+if (key) {
+    console.log(`Launching process with Pushbullet key: ${process.env.PUSHBULLET_KEY}`);
+    pusher = Promise.promisifyAll(new PushBullet(key));
+}
```

Comment:
The old code would throw an exception and stop.
I expect some error handling, as no activity does not indicate an issue.